### PR TITLE
feat(supabase): add supabase integration

### PR DESCRIPTION
**Pull request by tshedor**
Addresses #359 and #401

### CODE REVIEW

**Comment by Doetheman** [comment]
Supabase integration pretty dope. Flutterflow may be interested in using this

File: packages/brick_offline_first_with_supabase/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Unreleased
```
Comment (mateominato) [inline packages/brick_offline_first_with_supabase/CHANGELOG.md:1]: intended header?

**Review by mateominato — APPROVED** [review]
not to be confused with [superhot](https://superhotgame.com/)

**Review by Doetheman — APPROVED** [review]

File: packages/brick_offline_first_with_supabase/CHANGELOG.md (hunk 1 above)
Comment (tshedor) [reply packages/brick_offline_first_with_supabase/CHANGELOG.md:1]: Yeah it's sort of the always-present header that catches new stuff. It's a convention in all of the CHANGELOG files. I had a good reason for at some point but now couldn't tell you why

**Pull request merged**